internal/pkg/consensus/expected_test.go

```diff
@@ -161,7 +161,7 @@ func TestExpected_RunStateTransition_validateMining(t *testing.T) {
 		require.NoError(t, err)
 
 		blsMessages := make([][]*types.UnsignedMessage, tipSet.Len())
-		msg := types.NewUnsignedMessage(blsAddr, address.TestAddress2, 0, types.NewAttoFILFromFIL(0), "", []byte{})
+		msg := types.NewUnsignedMessage(blsAddr, address.TestAddress2, 0, types.NewAttoFILFromFIL(0), types.InvalidMethodID, []byte{})
```

Comment:
Blocking: The spec specifies the method id `0` for value transfers ("send"). `InvalidMethodID` is good for distinguishing an invalid id from one with meaning, but using it for sending is wrong. 

It looks like most uses of InvalidMethodID here are intended as sends, so renaming it is probably the easiest path.

- https://filecoin-project.github.io/specs/#systems__filecoin_vm__interpreter
- https://filecoin-project.github.io/specs/#systems__filecoin_vm__actor